sources/packages/backend/apps/queue-consumers/src/processors/schedulers/esdc-integration/ecert-integration/provincial-daily-disbursements-integration.scheduler.ts

```diff
@@ -0,0 +1,48 @@
+import { InjectQueue, Process, Processor } from "@nestjs/bull";
+import { DisbursementReceiptRequestService } from "@sims/integrations/esdc-integration";
+import { QueueService } from "@sims/services/queue";
+import { QueueNames } from "@sims/utilities";
+import { Job, Queue } from "bull";
+import { QueueProcessSummary } from "../../../models/processors.models";
+import { BaseScheduler } from "../../base-scheduler";
+import { DailyDisbursementReportQueueInDTO } from "../models/esdc";
+
+@Processor(QueueNames.FINProcessProvincialDailyDisbursementsIntegration)
+export class FINProcessProvincialDailyDisbursementsIntegrationScheduler extends BaseScheduler<DailyDisbursementReportQueueInDTO> {
+  constructor(
+    @InjectQueue(QueueNames.FINProcessProvincialDailyDisbursementsIntegration)
+    schedulerQueue: Queue<DailyDisbursementReportQueueInDTO>,
+    queueService: QueueService,
+    private readonly disbursementReceiptRequestService: DisbursementReceiptRequestService,
+  ) {
+    super(schedulerQueue, queueService);
+  }
+
+  /**
+   * Send provincial daily disbursement information to FIN.
+   * @params job job details.
+   * @returns Summary details of processing.
+   */
+  @Process()
+  async processFINProvincialDailyDisbursements(
+    job: Job<DailyDisbursementReportQueueInDTO>,
+  ): Promise<string> {
+    const summary = new QueueProcessSummary({
+      appLogger: this.logger,
+      jobLogger: job,
+    });
+    await summary.info(
+      `Processing FIN provincial daily disbursement integration job ${job.id} of type ${job.name}.`,
+    );
+    const batchRunDate = job.data.batchRunDate
+      ? new Date(job.data.batchRunDate)
+      : null;
+    await this.cleanSchedulerQueueHistory();
+    await summary.info(
+      `Completed FIN provincial daily disbursement integration job ${job.id} of type ${job.name}.`,
+    );
+    return this.disbursementReceiptRequestService.processProvincialDailyDisbursements(
```

Comment:
The method `processProvincialDailyDisbursements` must be executed before line 41.